esmgrids/mom_grid.py

```diff
@@ -98,8 +98,8 @@ def fromfile(cls, h_grid_def, v_grid_def=None, mask_file=None, calc_areas=True,
                 # These need to wrap around the globe. Copy ocn_area and
                 # add an extra column at the end. Also u-cells cross the
                 # tri-polar fold so add an extra row at the top.
-                area_ext = np.append(area[:], area[:, 0:1], axis=1)
-                area_ext = np.append(area_ext[:], area_ext[-1:, :], axis=0)
+                area_ext = np.append(area[:], np.fliplr(area[-1:, :]), axis=0)
+                area_ext = np.append(area_ext[:], area_ext[:, :1], axis=1)
```

Comment:
```suggestion
                area_ext = np.append(area[:], area[:, 0:1], axis=1)
                area_ext = np.append(area_ext[:], np.fliplr(area_ext[-1:, :]), axis=0)
```

everything else is column first, row second 🤷‍♂️

again i don't think this changes the answers due to symmetry ?

Reply:
The order does matter, but I think I always do it in the same order don't I?

Reply:
In these four lines:

https://github.com/COSIMA/esmgrids/blob/76f53e51a9d8c1ea310f0e21622ce62c949f6bad/esmgrids/mom_grid.py#L73-L78

the extra column is appended first, then the extra row is appended.

In the two lines of this comment, the row is done first, then the column. 

Feel free to ignore me, probably better not to break something which works

Reply:
Line 75 is operating on `dx` whereas line 78 is operating on `dy`

Reply:
hmm. Indeed!

Reply:
``` python
        # Copy ocn_area and add an extra row at the top, as u-cells cross the tri-polar fold.
        area_ext = np.append(area[:], np.fliplr(area[-1:, :]), axis=0)
        # These need to wrap around the globe. Add an extra column at the end 
        area_ext = np.append(area_ext[:], area_ext[:, :1], axis=1)
```

?

Reply:
Oh right yeah, the old comment is in the wrong order ta. I'll fix

Reply:
I've updated the comment in [6dc54c1](https://github.com/COSIMA/esmgrids/pull/25/commits/6dc54c187a14cca552b84517c40e3b06225d5ab6)